packages/e2e-tests/fixtures/blocks/core__legacy-widget.json

```diff
@@ -1,10 +1,14 @@
 [
     {
         "clientId": "_clientId_0",
-        "name": "core/legacy-widget",
+        "name": "core/missing",
```

Comment:
This is concerning, it looks like the `core/legacy-widget` block type isn't registered when tests run.